create-react-native-app/README.md

```diff
@@ -2,4 +2,4 @@
 
 ## Development
 
-`yarn && gulp` will start a watcher that will build artifacts and place them in the build directory.
+`yarn && npm start` will start a watcher that will build artifacts and place them in the build directory.
```

Comment:
`yarn start` also works here and I think it's better to recommend a consistent tool.

For this to work though, you'd need to rebase and get the new yarn.locks I just committed (I think).